RELEASE.md

```diff
@@ -117,6 +120,29 @@ Promotion should occur in two steps:
 
 **NOTE**: The promotion of wheels to PyPI can only be done once so take caution when attempting to promote wheels to PyPI, (see https://github.com/pypa/warehouse/issues/726 for a discussion on potential draft releases within PyPI)
 
+# Doing a Patch Release
+
+A patch release is a maintenance release of PyTorch that includes fixes for regressions found in a previous minor release.
+
+## Things to consider for a patch release
+
+1. Does the regression break core functionality?
+2. Does the regression break first party domain libraries that depend on locked versions of `pytorch`?
+    * [pytorch/vision](https://github.com/pytorch/vision)
+    * [pytorch/audio](https://github.com/pytorch/audio)
+    * [pytorch/text](https://github.com/pytorch/text)
+3. Is there not a viable workaround?
+
+## Patch release process
+
+1. Tag issues / pull requests with `patch release triage`
+    * ![image](https://user-images.githubusercontent.com/1700823/131175874-1526d098-d69c-48e4-ad75-c4afa1099f62.png)
+2. Patch release managers should triage all issues (on a bi-weekly basis) and submit issues into the related milestone (i.e. 1.9.1) if submitted issue match the patch release criteria
```

Comment:
what is the criteria?

Reply:
I believe we have cherry pick criteria that mostly fits the bill. Does it make sense to leverage that here @gchanan?

https://github.com/pytorch/pytorch/issues/51886

Reply:
Yes, I'm just unclear whether what you linked is considered the criteria or Patch Release Criteria is the criteria.

Reply:
@seemethere - Can we be more explicit and add the actual criteria?

Reply:
I've done that in an above section title `Patch Release Criteria`